src/plugins/stylesheet/PostCSSPlugin.ts

```diff
@@ -58,34 +58,29 @@ export class PostCSSPluginClass implements Plugin {
         file.bustCSSCache = true;
         file.loadContents();
 
-        let paths: string[] = this.options && this.options.paths || [];
-        paths.push(file.info.absDir);
+        let generateSourceMaps = true;
+        let postCssOptions;
+        let pathOptions: string[] = [file.info.absDir];
+
+        if (this.options) {
+            const {sourceMaps = generateSourceMaps, plugins, paths = pathOptions, ...otherOptions} = this.options;
+            generateSourceMaps = sourceMaps;
+            postCssOptions = otherOptions;
+            pathOptions = paths;
```

Comment:
I think `pathOptions` should always contain file.info.absDir no matter what, otherwise CSSDependencyExtractor will not work. Or am I missing something?

Reply:
You're not missing something, I set it up so it only contains `file.info.absDir` only if `paths` is not defined.
I'll fix this.